### PR TITLE
Complete till navbar

### PR DESCRIPTION
**Pull request by KBhautik6**
Screenshot 2024-01-15 at 10 28 01 AM

 - [x] Checking the code is duplicate or not?
 - [x] Is these code has bug
 - [ ] Is code is check by ankit sir or tara ma'am ?
 - [ ] Are you going to merge this or not?

### CODE REVIEW

File: src/stories/Navbar/Navbar.stories.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+export default {
+  title: 'Navbar',
+  component: Navbar,
+};
+
+export const ListNavbar = () => (
+  <div>
+    <Navbar>
+      <StyledNavbarSection>
+        <h2>Adorn Box</h2>
```
Comment (tarakhant1297) [inline src/stories/Navbar/Navbar.stories.js]: @KBhautik6 use typography

File: src/stories/Navbar/Navbar.stories.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        <StyledNavbarContentWrapper>
+          <div style={{ display: 'flex', alignItems: 'center' }}>
+            <StyledNavbarSearch
```
Comment (tarakhant1297) [inline src/stories/Navbar/Navbar.stories.js]: @KBhautik6 use input component with prefix search icon

File: src/theme.js [hunk 3]
```diff
@@ -1,3 +1,25 @@
+<<<<<<< HEAD
```
Comment (tarakhant1297) [inline src/theme.js]: @KBhautik6 remove this

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Navbar/Navbar.stories.js (hunk 1 above)
Comment (KBhautik6) [reply src/stories/Navbar/Navbar.stories.js]: okay ma'am

File: src/stories/Navbar/Navbar.stories.js (hunk 2 above)
Comment (KBhautik6) [reply src/stories/Navbar/Navbar.stories.js]: okay ma'am

File: src/theme.js (hunk 3 above)
Comment (KBhautik6) [reply src/theme.js]: okay ma'am

File: src/stories/Navbar/Navbar.stories.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+import {
+  StyledNavbarSection,
+  StyledNavbarContentWrapper,
+  StyledNavbarIconWrapper,
+  StyledNavbarLinkContainer,
+  StyledNavbarUnorderList,
+  StyledNavbarLists,
+  StyledNavbarListLink,
+} from "./Navbar.styled";
+import Button from "../Button/Button";
```
Comment (tarakhant1297) [inline src/stories/Navbar/Navbar.stories.js:12]: @KBhautik6 import Button from "../Button"; import must point to index file

File: src/stories/Navbar/Navbar.stories.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+import Badges from "../Badges/Badges";
+import Typography from "../Typography/Typography";
```
Comment (tarakhant1297) [inline src/stories/Navbar/Navbar.stories.js:14]: @KBhautik6 same same above 
import Badges from "../Badges/Badges";
import Typography from "../Typography/Typography";

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

**Review by tarakhant1297 — APPROVED** [review]

**Review by ankitadrixus08 — APPROVED** [review]

**Pull request merged**